sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosItemTest.java

```diff
@@ -179,11 +180,14 @@ public void queryItemsWithContinuationTokenAndPageSize() throws Exception{
         do {
             Iterable<FeedResponse<CosmosItemProperties>> feedResponseIterable =
                 feedResponseIterator1.iterableByPage(continuationToken, pageSize);
-            FeedResponse<CosmosItemProperties> feedResponse = feedResponseIterable.iterator().next();
-            int resultSize = feedResponse.getResults().size();
-            assertThat(resultSize).isEqualTo(pageSize);
-            finalDocumentCount += feedResponse.getResults().size();
-            continuationToken = feedResponse.getContinuationToken();
+            Iterator<FeedResponse<CosmosItemProperties>> feedResponse = feedResponseIterable.iterator();
+            for (; feedResponse.hasNext(); ) {
```

Comment:
while loop might be more suitable?

Reply:
yeah, it will be. Stupid intelliJ suggested this :P 
I will update it.